decentraland/unity-renderer
# Refactor/stickers controller QA

## QA 
This PR only exists in order to create a testing ground for QA
The stickers refactor changed how stickers are instantiated, but there should be no visible changes for them

Original PR: https://github.com/decentraland/unity-renderer/pull/3298
